src/index.spec.js

```diff
@@ -131,7 +131,7 @@ describe('Mongo', () => {
 
   describe('findUserById', () => {
     it('should return null for not found user', async () => {
-      const ret = await mongo.findUserById('unknowuser');
+      const ret = await mongo.findUserById('589871d1c9393d445745a57c');
```

Comment:
@pradel If the user is not found while you gave it syntactically good mongo id that's not an error in my opinion as it can, but giving it "invliad_id" as mongodb defines it, than you should get an error to let you know you are messing things up.
My question here is should we catch this error here and throw our AccountsError or should the accounts package be responsible for that?